### ISSUE TITLE
No visits being tracked

### ISSUE THREAD

**Issue by Fensterbank** [piwik/piwik#7306]
Since 2.11.1 (or 2.11.0 if it has something to do with https://github.com/piwik/piwik/issues/7290) no visits are beeing tracked.
Our piwik installation is running in a corporate intranet (accessible via http and https) to track intranet usage.
Piwik is hosted on a Windows Server 2012 with IIS 8.5.

Actions are tracked (but aren't visible anywhere), so the "Realtime Visit Counter" shows something like
> 0 visits and 14 actions in the last 3 minutes-

4 users (including me) reported the issue in this community thread: http://forum.piwik.org/read.php?2,124604

**Comment by jookk** [comment]
I have no actions recorded. piwik is set both to http and https with correct ssl cert. Tracked website is configured as piwik with site ssl. No visits since update to 2.11.1

**Comment by mattab** [comment]
Thanks for the reports. We need your help to debug this issue further as we cannot reproduce it yet. Can you please
- send by email to hello@piwik.org
- the following information: website URL, Piwik URL, Piwik login + password
- FTP or SSH information to the Piwik server files, so we can debug live on the server

We will investigate ASAP. THanks

**Comment by Fensterbank** [comment]
Thanks for the reply.
Sorry, I cannot provide this information, because the installation is in my customer's corporate intranet and there are no plans to make it accessible from outside. It's only for tracking internal SharePoint sites at the moment.
My two other piwik installations, aren't affected by this bug.

I hope @jookk or some other people concerned can provide you a direct access. :)

**Comment by jookk** [comment]
email sent. Hope, you find something

**Comment by DukeMini** [comment]
Same Problem for me too on Joomla 1.5 with actual generated tagging code, since update from 2.10.0 to 2.11.1

**Comment by mattab** [comment]
Hi Everyone,

can you please try upgrade to the latest Piwik beta: http://piwik.org/faq/how-to-update/faq_159/

This issue may be fixed.

If after upgrading to latest beta, you still have issue, please send us an email with all information necessary to connect to server and debug the issue (SSH or FTP)
Thanks

**Comment by jookk** [comment]
Thanks, I ll try in 30mins and will inform you. But I wrote that email 
previously.with requested details, so you can login to piwik and to ftp.


Dňa 2. marca 2015 6:07:42 používateľ Matthieu Aubry 
<notifications@github.com> napísal:

**Comment by mattab** [comment]
Hi @jookk thanks for sending the details, after you upgrade to latest beta I will try again on your server.

But also if someone else could send it, it would be great!! we need more info

**Comment by jookk** [comment]
I manually updated to beta and in administration was information: "WARNING: C:\xampp\htdocs\pw\vendor\doctrine\cache\lib\Doctrine\Common\Cache\PhpFileCache.php(75): Warning - include(): Failed opening 'C:\xampp\htdocs\pw\tmp\cache\tracker\eagercache-2112b1-ui.php' for inclusion (include_path='.;C:\xampp\php\PEAR') - Piwik 2.11.2-b1" and
"WARNING: C:\xampp\htdocs\pw\vendor\doctrine\cache\lib\Doctrine\Common\Cache\PhpFileCache.php(75): Warning - include(C:\xampp\htdocs\pw\tmp\cache\tracker\eagercache-2112b1-ui.php): failed to open stream: Permission denied - Piwik 2.11.2-b1".

**Comment by mattab** [comment]
I cannot use @jookk details to access the server, there seems to be a problem. Therefore I still one of you to send me access via SSH or FTP. 

Thank you in advance (otherwise we can't fix this bug)

**Comment by jookk** [comment]
You cant login to ftp or piwik tracker?


Dňa 2. marca 2015 7:14:49 používateľ Matthieu Aubry 
<notifications@github.com> napísal:

**Comment by mattab** [comment]
I can login but when I change files on the FTP this is not applied to
the live piwik server

**Comment by jookk** [comment]
should be fixed now (ftp issues)

**Comment by mattab** [comment]
To anyone who has this issue, can you add in your config/config.ini.php
```
[Tracker]
debug=1
```

then paste here a website being tracked in your website?

**Comment by mattab** [comment]
@jookk FTP issues not fixed (see email)

**Comment by jookk** [comment]
so, if you name files, which I have to send to you to edit, I ll send it and then after your edit I rewrite files on my piwik install, then you can test again. (if you still want to...) Sorry for my blacky ftp instance...

**Comment by mattab** [comment]
I'm closing issue since the two users who privately emailed me now have their problem solved. As far as we know, there is no bug. if you have a problem with visits not tracked, please open new issue

**Comment by mattab** [comment]
FYI: the third person who emailed me has a new issue. When I got to piwik.php?xyz I get the following error:
```
Error query: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '73360' for key 'PRIMARY' In query: INSERT INTO piwik_log_visit (idvisitor, config_id, location_ip, location_longitude, location_latitude, location_region, visitor_localtime, location_country, location_city, config_device_type, config_device_model, config_os, config_os_version, visit_total_events, visitor_days_since_last, config_quicktime, config_pdf, config_realplayer, config_silverlight, config_windowsmedia, config_java, config_gears, config_resolution, location_provider, config_cookie, config_director, config_flash, config_device_brand, config_browser_version, visitor_returning, visitor_days_since_order, visitor_count_visits, visit_entry_idaction_name, visit_entry_idaction_url, visit_first_action_time, visitor_days_since_first, visit_total_time, visit_goal_buyer, visit_goal_converted, visit_last_action_time, visit_exit_idaction_name, visit_exit_idaction_url, referer_url, location_browser_lang, config_browser_engine, config_browser_name, referer_type, referer_name, visit_total_actions, visit_total_searches, referer_keyword, idsite) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?) Parameters: array ( 0 => '�����Sg', 1 => '��3on=��', 2 => '$0' . "\0" . '' . "\0" . '�' . "\0" . '' . "\0" . '' . "\0" . '' . "\0" . '' . "\0" . '' . "\0" . '' . "\0" . '' . "\0" . '' . "\0" . '', 3 => '117.281', 4 => '31.864', 5 => '01', 6 => '13:23:44', 7 => 'cn', 8 => 'Hefei', 9 => 0, 10 => '', 11 => 'UBT', 12 => '', 13 => 0, 14 => '0', 15 => 0, 16 => 0, 17 => 0, 18 => 0, 19 => 1, 20 => 0, 21 => 0, 22 => '1920x1080', 23 => 'Ip', 24 => 1, 25 => 0, 26 => 1, 27 => '', 28 => '36.0', 29 => 0, 30 => 0, 31 => 1, 32 => 856, 33 => 904, 34 => '2015-03-04 00:24:45', 35 => '0', 36 => 0, 37 => 0, 38 => 0, 39 => '2015-03-04 00:24:45', 40 => 856, 41 => 904, 42 => '', 43 => 'en', 44 => 'Gecko', 45 => 'FF', 46 => 1, 47 => '', 48 => 1, 49 => 0, 50 => '', 51 => 2, )

```

I don't even understand how this error could occur since the PRIMARY key is `idvisit` and this field is not specified in the query. Maybe the table is corrupted or something? Anyway, this is not a Piwik bug.

**Comment by mattab** [comment]
Regarding my previous comment here is user's replay:

> I was able to solve the problem – as far as I can see.
> Corrupt tables:
>               piwik_log_visit
> piwik_log_link_visit_action
> I did a quick repair with phpMyAdmin and now visits are tracked again.

**Comment by Fensterbank** [comment]
A repair did not help for me, but according to phpMyAdmin, the tables didn't seem to be corrupt.
We restored an old database backup from January (message to me: Make a scheduled database backup!) and let piwik do the database upgrade from 2.9 to 2.11.2.
Now tracking works!

I don't know, what occured the problem, it must have something to do with the database upgrade.
By accepting to lose data, the issue is now solved/workarounded.
